test/lsp/lsp_formatting.ml

```diff
@@ -23,7 +23,9 @@ let doc = {cobol|
   |cobol};;
 
 let%expect_test "simple-formatting-request" =
-  let { projdir; end_with_postproc }, server = make_lsp_project () in
+  let { projdir; end_with_postproc }, server =
+    make_lsp_project ~toml:"source-format = \"free\"\n" ()
```

Comment:
This could have deserved a little `make_lsp_free_format_project` shortcut to avoid duplication.

Reply:
It could, but I believe test code deserves to be as plain as possible, hence I did not do that -- in fact there is much code duplication between the test cases already.